src/hotspot/share/prims/jvmtiEnv.cpp

```diff
@@ -928,6 +928,8 @@ JvmtiEnv::GetAllThreads(jint* threads_count_ptr, jthread** threads_ptr) {
 jvmtiError
 JvmtiEnv::SuspendThread(jthread thread) {
   JavaThread* current = JavaThread::current();
+  HandleMark hm(current);
+  Handle self_tobj = Handle(current, nullptr);
```

Comment:
This doesn't have to have the assignment, which the compiler will optimize away anyway.

Handle self_tobj;

does the same thing.

Reply:
Both local variables java_thread and thread_oop should be declared before line 939 so we don't have to worry about the oop being unhandled and java_thread is only used locally.

Reply:
Thank you, Coleen. Fixed now.